### DISCUSSION TITLE
It'd be nice to be able to rerun `not allowed` workflows

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#82187] [Actions]
### Select Topic Area

Product Feedback

### Body

https://github.com/jsoref/check-spelling/actions/runs/7331405771

<img width="1337" alt="image" src="https://github.com/community/community/assets/2119212/8f58ded8-2ad8-4036-9c1b-37fe39c4c7de">

I can fix this by adding the required allow item, but once I've done this, I can't retrigger the run -- I have to make another commit. I'd really like to skip making another commit just to resolve this. 

If I fail to define a secret or variable for my repository and trigger a workflow, it'll fail, but I can fix it and rerun. I don't see that as any different from this.

**Comment by Imadnajam** [comment]
I would like to propose the option to rerun workflows without requiring an extra commit in cases where the only modification is the addition of an "allow" item or similar adjustments.
